man/systemd.netdev.xml

```diff
@@ -1979,6 +1982,77 @@
     </variablelist>
   </refsect1>
 
+  <refsect1>
+    <title>[Batadv] Section Options</title>
+    <para>The [Batadv] section only applies for
+    netdevs of kind <literal>batadv</literal> and accepts the
+    following keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>GatewayMode=</varname></term>
+        <listitem>
+          <para>The GatewayMode is a string of either "off", "server" or "client"</para>
```

Comment:
Use `<option>off</option>`, etc, so it'll get listed in the index. Missing period.